pootle/apps/pootle_app/management/commands/create_project.py

```diff
@@ -0,0 +1,149 @@
+            action="store",
+            dest="mapping",
+            help="Custom mapping layout",
+        )
+        parser.add_argument(
+            "--fs-type",
+            action="store",
+            dest="fs_type",
+            default="localfs",
+            choices=fs_plugins.gather().keys(),
+            help="Filesystem type: {0}".format(
+                " | ".join(fs_plugins.gather().keys())),
+        )
+        parser.add_argument(
+            "--fs-url",
+            action="store",
+            dest="fs_url",
+            default="",
+            help="Filesystem url.",
+        )
+        parser.add_argument(
+            "--source-language",
+            action="store",
+            dest="sourcelang",
+            default="en",
+            help="Source language. Examples: [en | es | fr | ...]",
+        )
+        parser.add_argument(
+            "--report-email",
+            action="store",
+            dest="contact",
+            help="Contact email for reports. Example: admin@mail.com",
+        )
+        parser.add_argument(
+            "--disabled",
+            action="store_true",
+            dest="disabled",
+            help="Does the project start disabled?",
+        )
+        parser.add_argument(
+            "--checkstyle",
+            action="store",
+            dest="checkstyle",
+            choices=PROJECT_CHECKERS.keys(),
+            default="standard",
+            help="Quality check styles. Example: {0}"
+                 .format(" | ".join(PROJECT_CHECKERS.keys())),
+        )
+
+    def handle(self, **options):
+        self.create_project(**options)
+
+    def create_project(self, **options):
+        """Immitates the behaviour of the admin interface
+            for creating a project from command line."""
+
+        try:
+            source_language_obj = Language.objects.get(code=options["sourcelang"])
+        except Language.DoesNotExist:
+            raise CommandError("Source language %s does not exist"
+                               % options["sourcelang"])
+
+        if(options["contact"]):
+            try:
+                validate_email(options["contact"])
+            except ValidationError as e:
+                raise CommandError(e)
+
+        new_project = Project.objects.create(
+            code=options["code"],
+            fullname=options["name"] or options["code"].capitalize(),
+            checkstyle=options["checkstyle"],
+            source_language=source_language_obj,
+            filetypes=options["filetypes"] or ["po"],
+            report_email=options["contact"] or "",
+            disabled=options["disabled"])
+
+        if(options["preset_mapping"]):
+            new_project.config["pootle_fs.translation_mapping"] = dict(
+                default=FS_PRESETS[options["preset_mapping"]][0])
+        else:
+            new_project.config["pootle_fs.translation_mapping"] = dict(
+                default=options["mapping"])
+
+        if(options["fs_type"]):
```

Comment:
We have a default for this argument, so this check is unnecessary, just assign it.